pkg/service/service.go

```diff
@@ -60,6 +60,22 @@ type monitorNotify interface {
 	SendNotification(msg monitorAPI.AgentNotifyMessage) error
 }
 
+// Name represents the fully-qualified reference to the service by name, including both the
+// namespace and name of the service.
+type Name struct {
```

Comment:
`type Name struct` looks too broad. Maybe `type ServiceName struct` instead?

Reply:
I had it like that, but then when using from other packages `service.Name` felt much better than `service.ServiceName`. What do you think?

Reply:
Sounds good

Reply:
So kept as-is.